isis/appdata/import/cassis.lbl.tpl

```diff
@@ -1,151 +1,421 @@
+{% set PO=Product_Observational %}
+{% set CassHeader="Product_Observational.CaSSIS_Header" %}
+{% set IdArea="Product_Observational.Identification_Area" %}
+{% set FileArea="Product_Observational.File_Area_Observational" %}
+{% set threeDArray="Product_Observational.File_Area_Observational.Array_3D_Image" %}
+
 Object = IsisCube
   Object = Core
-    StartByte   = 65537
-    Format      = Tile
-    TileSamples = 512
-    TileLines   = 280
 
+    {% if exists(threeDArray) %}
+    Group = Dimensions
+      Samples = {{ PO.File_Area_Observational.Array_3D_Image.Axis_Array.0.elements }}
+      Lines   = {{ PO.File_Area_Observational.Array_3D_Image.Axis_Array.1.elements }}
+      Bands   = {% if exists(FileArea + ".Array_3D_Image.Axis_Array.2.elements") %}
+                {{ PO.File_Area_Observational.Array_3D_Image.Axis_Array.2.elements }}
+                {% else %}
+                1
+                {% endif %}
+    End_Group
+
+    Group = Pixels
+      {% if exists("Product_Observational.File_Area_Observational.Array_3D_Image.Element_Array.idl_data_type") %}
```

Comment:
idl_data_type strangely does not exist in reingested cassis images. I have no idea how this is being handled correctly in tgocassis2isis. Therefore I had to put a if statement to default to "Real" for this instance.

Reply:
@kberryUSGS, @amystamile-usgs, We probably don't want to track down where it disappeared, so the default is good for this round. If we find out it was just not transferred at some stage, then we can fix it later. Any objections?